doc/releases/release-notes-4.2.rst

```diff
@@ -69,6 +69,11 @@ New APIs and options
 
     * :c:func:`bt_le_get_local_features`
 
+* Display
```

Comment:
@fabiobaltieri @danieldegrasse @jfischer-no 

Overall, I don't think there's any problem.
Could we hear your opinions?